Documentation/Setup/Config/Index.rst

```diff
@@ -671,14 +682,32 @@ disableAllHeaderCode
          false
 
    Description
-         If this is set, none of the features of the :ref:`page` object is processed
-         and the content of the page will be the result of the cObject array
-         (1,2,3,4...) of the :ref:`page` object. This means that the result of the
-         cObject should include everything from the :html:`<HTML>` .... to the :html:`</HTML>`
-         tag!
+         If this is not set or set to false the :ref:`page` object automatically
+         outputs a HTML skeleton, see :ref:`page_output`.
+
+         To disable this default behaviour set :ts:`disableAllHeaderCode = 1`.
+         The page outputs only the result of the cObject array
+         (1,2,3,4...) of the :ref:`page` object.
 
          Use this feature in templates supplying other content-types than HTML.
-         That could be an image or a WAP-page!
+         That could be an image, a WAP-page, an ajax request or a JSON response.
```

Comment:
can we remove the mentioning of WAP pages? I think their time is past and your other examples are much better (maybe mention RSS or XML)